pkg/environment/flags.go

```diff
@@ -58,6 +59,8 @@ func InitFlags(fs *flag.FlagSet) {
 	fs.StringVar(f, "feature", "", "run only Features matching `regexp`")
 
 	fs.StringVar(ipFilePath, "images.producer.file", "", "file path for file-based image producer")
+
+	fs.BoolVar(skipCleanupOnFail, "skip-cleanup-on-fail", true, "Set this flag to skip cleanup if test fails.")
```

Comment:
Let's simplify this flag:

```suggestion
	fs.BoolVar(cleanupOnFail, "cleanup-on-fail", false, "Set this flag to cleanup even if test fails")
```